src/handlers.py

```diff
@@ -159,14 +161,15 @@ def send_to_dojo(body, meta, logger, **_):
         logger.debug(data)
 
         try:
-            response: requests.Response = requests.post(
+            response: httpx.Response = httpx.post(
                 settings.DEFECT_DOJO_URL + "/api/v2/reimport-scan/",
                 headers=headers,
                 data=data,
                 files=report_file,
                 verify=True,
-                proxies=proxies,
+                mounts=proxy_mounts,
```

Comment:
`mounts` is only valid on httpx.Client, not on httpx.post()

I'd either

- simplify to httpx.post(..., proxies=proxies)
- or wrap everything in a client

```py
with httpx.Client(mounts=proxy_mounts) as client:
    response = client.post(...)
```